projects/ccf-rui/src/app/modules/info/info-panel/info-panel.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule, SecurityContext } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { InfoPanelComponent } from './info-panel.component';
+import { MarkdownModule } from 'ngx-markdown';
```

Comment:
You need to add ngx-markdown to package.json/package-lock.json (build is failing)

Reply:
Ah I wonder why this didn't happen. I did but the `--save` tag. I'll fix it.